src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/MarkDuplicatesDataflowIntegrationTest.java

```diff
@@ -40,5 +48,25 @@ public void testMarkDuplicatesDataflowIntegrationTestLocal(final File input, fin
         Assert.assertEquals(new XReadLines(outputFile).readLines().size(), totalExpected);
 
         Assert.assertEquals(new XReadLines(outputFile).readLines().stream().filter(line -> line.contains("duplicateFragment\":true")).count(), dupsExpected);
+  }
+
+
+  @Test(groups = "dataflow", dataProvider = "md_optical")
+    public void testOpticalDuplicateDetection(final File input, final long totalExpected, final long dupsExpected) throws IOException {
```

Comment:
Currently that number is not actually output anywhere - that is why we need metrics.